Marlin/src/HAL/HAL_DUE/u8g_com_HAL_DUE_st7920_sw_spi.cpp

```diff
@@ -57,42 +57,26 @@
 
 #include "../../inc/MarlinConfigPre.h"
 
-#if HAS_GRAPHICAL_LCD
+#if ENABLED(U8GLIB_ST7920)
+
+#include "u8g_com_HAL_DUE_sw_spi_shared.h"
 
-#include "../shared/Marduino.h"
 #include "../shared/Delay.h"
 
 #include <U8glib.h>
 
-#include "u8g_com_HAL_DUE_sw_spi_shared.h"
-
-Pio *SCK_pPio, *MOSI_pPio;
-uint32_t SCK_dwMask, MOSI_dwMask;
-
-static void spiSend_sw_DUE(uint8_t val) { // 800KHz
-  for (uint8_t i = 0; i < 8; i++) {
-    if (val & 0x80)
-      MOSI_pPio->PIO_SODR = MOSI_dwMask;
-    else
-      MOSI_pPio->PIO_CODR = MOSI_dwMask;
-    DELAY_NS(48);
-    SCK_pPio->PIO_SODR = SCK_dwMask;
-    DELAY_NS(905); // 762 dead, 810 garbage, 858/0 900kHz, 905/1 825k, 953/1 800k, 1000/2 725KHz
-    val <<= 1;
-    SCK_pPio->PIO_CODR = SCK_dwMask;
-  }
-}
```

Comment:
If the intention is to have an 800MHz SPI and also a 3.5MHz SPI, but not at the same time, then we could add conditionals to the shared one to get the different delays.